osf/models/preprint_provider.py

```diff
@@ -31,6 +31,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     share_source = models.CharField(blank=True, max_length=200)
     allow_submissions = models.BooleanField(default=True)
     additional_providers = fields.ArrayField(models.CharField(max_length=200), default=list, blank=True)
+    preprint_word = models.TextField(default='preprint')
```

Comment:
Just a passing comment: Seems like the `choices` option could be used here? See https://docs.djangoproject.com/en/1.11/ref/models/fields/#choices.

Also, `CharField` over `TextField` since this field won't be holding large amounts of text?